exec/exec.go

```diff
@@ -79,3 +82,73 @@ func Run(ctx context.Context, command string, arg []string) (*CommandExecutor, e
 
 	return commandExecutor, nil
 }
+
+// Output returns stdout, stderr and error status for a given shell command
+func Output(ctx context.Context, command []string, timeout ...string) ([]byte, []byte, error) {
```

Comment:
can we change the signature to take a `command ...string`? it looks nicer then array of strings